src/Bling.js

```diff
@@ -191,7 +191,17 @@ class Bling extends Component {
          *
          * @property style
          */
-        style: PropTypes.object
+        style: PropTypes.object,
+        /**
+         * An optional property to control non-personalized Ads.
+         * https://support.google.com/admanager/answer/7678538
+         *
+         * Set to `true` to mark the ad request as NPA, and to `false` for ad requests that are eligible for personalized ads
+         * It is `false` by default, according to Google's definition.
+         *
+         * @property npa
+         */
+        npa: PropTypes.bool
```

Comment:
nit=pick, but according to gpt doc https://developers.google.com/doubleclick-gpt/reference#googletag.PubAdsService_setRequestNonPersonalizedAds 

npa should be a number 0 or 1, not a boolean 

```
npa: PropTypes.oneOf([0, 1])
```

otherwise LGTM, maybe @cwelch5 or @miblanchard or @LGCambra  will merge it for you 😸

Reply:
Also a good point.  
But if you take a look at line [814](https://github.com/nfl/react-gpt/pull/97/files#diff-c11a8b813fb0c6b0b810b426e416a6e5R814) you will see that I'm converting it to `Number` before pass it to `pubadsProxy`